core/tests/protractor_desktop/skillEditor.js

```diff
@@ -105,19 +105,27 @@ describe('Skill Editor functionality', function() {
   });
 
   it('should edit rubrics for the skill', function() {
-    skillEditorPage.expectRubricExplanationToMatch(0, 'Explanation 0');
-    skillEditorPage.expectRubricExplanationToMatch(1, 'Explanation 1');
-    skillEditorPage.expectRubricExplanationToMatch(2, 'Explanation 2');
-
-    skillEditorPage.editRubricExplanationWithIndex(0, 'Explanation 0 edited');
-    skillEditorPage.editRubricExplanationWithIndex(1, 'Explanation 1 edited');
-    skillEditorPage.editRubricExplanationWithIndex(2, 'Explanation 2 edited');
+    skillEditorPage.expectRubricExplanationsToMatch(0, ['Explanation 0']);
+    skillEditorPage.expectRubricExplanationsToMatch(
+      1, ['Skill 1', 'Explanation 1']);
+    skillEditorPage.expectRubricExplanationsToMatch(2, ['Explanation 2']);
+
+    skillEditorPage.editRubricExplanationWithIndex(
+      0, 0, 'Explanation 0 edited');
+    skillEditorPage.editRubricExplanationWithIndex(
+      1, 1, 'Explanation 1 edited');
+    skillEditorPage.deleteRubricExplanationWithIndex(1, 0);
+    skillEditorPage.editRubricExplanationWithIndex(
+      2, 0, 'Explanation 2 edited');
```

Comment:
We don't seem to ever test editing multiple explanations for the same difficulty. Since that's the new functionality added by this PR, could you add tests to cover that?

Reply:
Done, added new explanations here and edited it.